app/build.gradle

```diff
@@ -32,6 +32,8 @@ dependencies {
 
     implementation "com.facebook.fresco:fresco:$frescoVersion"
     implementation "com.facebook.fresco:animated-gif:$frescoVersion"
+    implementation "com.camerakit:camerakit:$cameraKitVersion"
+    implementation "com.camerakit:jpegkit:0.1.0"
```

Comment:
Should this version number go into the list of version numbers?

Reply:
yeah this should. TBH I don't know what this is for though. I should look into that first.

Reply:
needs more jpg

Reply:
it's used in post processing of the raw image to jpeg compression. Keeping it in and parameterizing the version